onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -1130,7 +1129,8 @@ Status MIGraphXExecutionProvider::Compile(const std::vector<FusedNodeAndGraph>&
             if (mgx_type != mgx_s.type()) {
               LOGS_DEFAULT(FATAL) << "MIGraphX: param type mismatch";
             }
-            m.add(name, migraphx::argument(param_shapes[name], input_tensor.GetTensorMutableRawData()));
+            m.add(name, migraphx::argument(param_shapes[name],
+                                           const_cast<void*>(input_tensor.GetTensorRawData())));
```

Comment:
It's a shame this const_cast can't be avoided (KernelContext::GetInput() must return a const/unowned input).

Reply:
It is not about our interfaces. It is about ROCM.

Reply:
Right. That's what I mean. Our interface is correctly returning const, but this function is not specifying const.